examples/simple-spark-cluster.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: spark.stackable.tech/v1alpha1
 kind: SparkCluster
 metadata:
-  name: simple-spark
+  name: spark
```

Comment:
Is this a leftover from testing? Otherwise, do we want to remove the `simple-` prefix in all operator examples?

Reply:
I kind of left it as proof that the bug is fixes but now that you ask, I Iike it better than "simple-spark and woud indeed remove the "simple" prefix in the examples.

Reply:
Hm.. I think there can still be value in saying that this is an MVP example, as opposed to more complex setups (multiple rolegroups, complex integrations with other systems, specific tuning, etc).

Reply:
yeah i saw that its fixed and like the examples to look the same. The `simple-spark` is ok i guess. It is simple after all since we basically set no config options.

Reply:
reverted.